diff -Naur v2ftp2D/v2ftp2D.C v2ftp2D/v2ftp2D.C
--- v2ftp2D.C	
+++ v2ftp2D.C	
@@ -963,14 +963,14 @@
 }
 
 // Term that is directly added to the momentum equation
-tmp<fvVectorMatrix> v2ftp2D::divDevReff(volVectorField& U) const
+tmp<fvVectorMatrix> v2ftp2D::divDevReff() const
 {
     return
     (
        fvc::grad(phiReal())
      + fvc::curl(psiReal())
-     + fvc::laplacian(nut_, U, "laplacian(nuEff,U)")
-     - fvm::laplacian(nuEff(), U)
+     + fvc::laplacian(nut_, U_, "laplacian(nuEff,U_)")
+     - fvm::laplacian(nuEff(), U_)
     );
 }
 
diff -Naur turbulentPotential/v2ftp2D.H turbulentPotentialFixU/v2ftp2D.H
--- v2ftp2D.H
+++ v2ftp2D.H
@@ -401,7 +401,7 @@
         virtual tmp<volSymmTensorField> devReff() const;
 
         //- Return the source term for the momentum equation
-        virtual tmp<fvVectorMatrix> divDevReff(volVectorField& U) const;
+        virtual tmp<fvVectorMatrix> divDevReff() const;
 
         //- Solve the turbulence equations and correct the turbulence viscosity
         virtual void correct();
